### DISCUSSION TITLE
surebet

### DISCUSSION THREAD

**Discussion by kreon76** [pretrehr/Sports-betting#23] [General]
Bonjour,

J'aimerais juste savoir si il est prévu dans une mise à jour prochaine un onglet spécial avec les surebets et les TRJ des matchs ?

Si cela n'est pas prévu ce n'est pas grave du tout c'est juste pour savoir.

Merci par avance

PS : si cette option existe en ligne de commande sans l'interface graphique je suis très intéressé ;o)

Cdt

**Comment by pretrehr** [comment]
Bonjour,
Alors ce n'était pas forcément prévu mais c'est vrai que c'est toujours intéressant de savoir ce genre d'informations et ce n'est pas très compliqué à rajouter donc je peux ajouter un onglet supplémentaire assez rapidement.
En attendant, vous pouvez toujours utiliser l'onglet "Pari simple", en mettant une mise de 1 et une cote minimale de 1.01 et regarder pour chaque bookmaker (et chaque sport) si vous avez un match qui génère une plus-value positive. Le TRJ sera alors donné par la formule :
```
1 + plus-value affichée
```
C'est un peu fastidieux mais ça fait le job...

Sinon, en ligne de commande, je vous conseille de regarder les différentes fonctions du fichier basic_functions.py.
Ce sont des fonctions vraiment simples mais qui automatisent les calculs les plus récurrents :
Notamment pour calculer le TRJ de plusieurs cotes, vous pouvez utiliser la fonction 
`gain([cote1, coteN, cote2])`

**Reply by kreon76** [reply]
Je ne veux pas vous donner + de travail ;o) ce sont des infos intéressantes je trouve mais je peux les trouver dans Pari simple c'est vrai.

**Reply by kreon76** [reply]
J'ai fait une synchro et qu'est ce que je vois ? L'onglet Surebet !! Que dire ? à part Grand MERCI !
